lambdas/common/utils/ecs_utils.py

```diff
@@ -157,3 +157,32 @@ def describe_service(ecs_client, cluster_arn, service_arn):
         cluster=_name_from_arn(cluster_arn),
         services=[_name_from_arn(service_arn)]
     )['services'][0]
+
+
+def run_task(
+        ecs_client,
+        cluster_name,
+        task_definition,
+        started_by,
+        container_name="app",
+        command=[]):
```

Comment:
I’d suggest adding a `overrides` parameter to this function, and adding its value to the existing `overrides` in the RunTask API – some tasks require a bit of configuration (e.g. environment variables), and that’s quite a flexible way to do it.

Reply:
I think there's possibly utility in keeping the interface as simple as possible here. I'd prefer to update this as required.